extensions/amp-story/1.0/amp-story.js

```diff
@@ -833,6 +834,16 @@ export class AmpStory extends AMP.BaseElement {
           accessService.areFirstAuthorizationsCompleted();
       accessService.onApplyAuthorizations(
           () => this.onAccessApplyAuthorizations_());
+
+      const firstPage = this.pages_[0].element;
+      // First amp-story-page can't be paywall protected.
+      // Throws an error during development, but just ignore the amp-access rule
```

Comment:
Is this still true for user()?

Reply:
Yup, this is intended for the publishers, whereas `dev` is meant for AMP core developers.